library/src/main/java/com/mapzen/android/MapzenMap.java

```diff
@@ -280,16 +309,30 @@ public LngLat coordinatesAtScreenPosition(double screenX, double screenY) {
     /**
      * Set tap responder for tap gestures on map.
      */
-    public void setTapResponder(final TouchInput.TapResponder tapResponder) {
-        this.tapResponder = tapResponder;
-        mapController.setTapResponder(this.tapResponder);
+    public void addTapResponder(final TouchInput.TapResponder tapResponder) {
```

Comment:
I'm still not sure supporting multiple gesture listeners of the same type is worth the added complexity. If we decide to go this route we will need to give the developer a way to remove a single listener as well as to clear all.

For comparison, Google Maps only allows a single listener of each type. And clearing the listener is achieved by calling the same method and passing `null` as the listener param.

https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap.html#setOnMapClickListener(com.google.android.gms.maps.GoogleMap.OnMapClickListener)